Makefile

```diff
@@ -6,7 +6,7 @@ TARGET := kube-vip
 .DEFAULT_GOAL: $(TARGET)
 
 # These will be provided to the target
-VERSION := 0.2.2
+VERSION := 0.2.3
```

Comment:
Is this a patch level change or more significant?

Reply:
I'm making versioning up as I go along